javascript/core/intro-js/javascript-basics-part-1.md

```diff
@@ -33,9 +33,17 @@ Like most programming languages, JavaScript has a number of rules its code must
 
 Everything in JavaScript is case sensitive.
 
+```javascript
+var password;
+// Is not the same as
+var passWord;
+/or
```

Comment:
this should have a double forward slash "//or"

Reply:
Yes, and apologies, I was doing markdown on my phone. Good catch.

Reply:
It's not a problem, I just wanted to point these out before we could merge. Thank you for taking the time to improve the content.